### PR TITLE
APIs creates

### PR DESCRIPTION
**Pull request by arejandro**
I made POST APIs for Student accounts and Job Postings.

### CODE REVIEW

File: src/pages/api/student/[email]/index.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { updatePasswordByEmail } from "@/backend/actions/student";
```
Comment (LorenzCastillo) [inline src/pages/api/student/[email]/index.ts:1]: Same thing here, just for organizing purposes to maybe add what's a 3rd Party Import and what's a Local Import

File: src/pages/api/student/create.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { EMAIL_REGEX, PASSWORD_REGEX, PHONE_REGEX, STUDENTID_REGEX } from "@/shared/regex";
+
```
Comment (LorenzCastillo) [inline src/pages/api/student/create.ts:3]: Here too

**Review by LorenzCastillo — APPROVED** [review]
Looks like it all checks out! I'm just pretty picky with consistency tho so if you'd like you can make changes on what I commented. Otherwise everything looks good!

**Review by emilsangil — APPROVED** [review]
code looks good to me

**Pull request merged**